offchain/dusd-ui/netlify.toml

```diff
@@ -0,0 +1,17 @@
+[[headers]]
+	for = "*.avif"
+	[headers.values]
+		Content-Type = "image/avif"
+		Content-Disposition = "inline"
+
+[[headers]]
+	for = "*.heif"
+	[headers.values]
+		Content-Type = "image/heif"
+		Content-Disposition = "inline"
```

Comment:
do we need these?

Reply:
Future-proofing. AVIF would be great to have if we have raster-style images like the "team" section with photographic portraits. HEIF is another image container that could be useful.